src/CompilerHost.ts

```diff
@@ -48,7 +48,7 @@ export class CompilerHost
   private readonly tsHost: ts.WatchCompilerHostOfConfigFile<
     ts.EmitAndSemanticDiagnosticsBuilderProgram
   >;
-  private lastProcessing?: Promise<ts.Diagnostic[]>;
+  protected lastProcessing?: Promise<ts.Diagnostic[]>;
```

Comment:
There's lots of `private` becoming `protected` - I'm guessing there's an overall reason for that but I don't know what it is..  could you enlighten me please? 😄

Reply:
Some tests access these properties. You can still access `private` or `protected` properties by accessing them via `instance['propertyname' ]` instead of `instance.propertyname` from the tests, but if they are `private`, typescript emits them as `any` - if you make them `protected`, typescript emits the correct type and the tests can access that correct type then.